robotis-op2/robot-window.md

```diff
@@ -0,0 +1,18 @@
+# Robot Window
+
+When you double click on the robot, a new window appears. This window is called
+`Robot Window`, it has several tabs allowing you to perform different things.
+First tabs concern the simulation. They will be described here, the last tab is
+used to interact with the real robot and will therefore be described in next
+sections.
+
+You can at any time restart the `Robot Window` by pressing the revert button of
```

Comment:
Yes. I was thinking about a user reading that and saying to himself: "This is a professional software, and it seems they know it can freeze whereas it should not badly behave..."
So I removed it but I let the information that it can be restarted (the user will refer to this point in the unlikely case something is going wrong).
What do you recommend to do ?